homeassistant/components/image_processing/__init__.py

```diff
@@ -64,6 +64,24 @@
 PLATFORM_SCHEMA_BASE = cv.PLATFORM_SCHEMA_BASE.extend(PLATFORM_SCHEMA.schema)
 
 
+def draw_box(draw, box, img_width, img_height, text="", color=(255, 255, 0)):
```

Comment:
`draw` is an instance of `ImageDraw` from `PIL`, which isn't a hard dependency on the core Image Processing component.  If we want to move this out of the individual integrations, it probably should be.

Reply:
PIL itself provides a draw utility, is there a reason we don't use it? https://pillow.readthedocs.io/en/3.1.x/reference/ImageDraw.html#PIL.ImageDraw.PIL.ImageDraw.Draw.rectangle

Reply:
image processing is not a core integration, we can add it there to manifest...

Reply:
future, you don't need to import the function. You can use `hass.components.image_processing.draw_box()`